cn_indian_payroll/cn_indian_payroll/overrides/salary_slip.py

```
@@ -21,18 +21,13 @@
 	money_in_words,
 	rounded,
 )
-from datetime import datetime
 
+from datetime import datetime, timedelta
```

Ruff (0.8.2)

Redefinition of unused `datetime` from line 2

(F811)